diff.py

```diff
@@ -1961,7 +1963,29 @@ def process(dump: str, config: Config) -> List[Line]:
             row_parts = [part.lstrip() for part in row.split(" ", 1)]
         mnemonic = row_parts[0].strip()
         args = row_parts[1] if len(row_parts) >= 2 else ""
-        row = mnemonic + "\t" + args.replace("\t", "  ")
+
+        if arch.name == "ppc":
+            if mnemonic == "li":
+                mnemonic = "addi"
+                args_parts = args.split(",")
+                args = args_parts[0] + ",0," + args_parts[1]
+            if mnemonic == "lis":
+                mnemonic = "addis"
+                args_parts = args.split(",")
+                args = args_parts[0] + ",0," + args_parts[1]
```

Comment:
This needs a comment. Also, can we do it just if there's a successive R_PPC_EMB_SDA21?

Reply:
hmmmm, I guess you are right!